Update custom-domain.md (#116278)
Azure/azure-content: articles/active-directory-b2c/custom-domain.md

This article describes how to enable custom domains in your redirect URLs for Azure Active Directory B2C (Azure AD B2C). By using a verified custom domain, you've benefits such as: 

- It provides a more seamless user experience. From the user's perspective, they remain in your domain during the sign in process rather than redirecting to the Azure AD B2C default domain *&lt;tenant-name&gt;.b2clogin.com*.

> -  Staying in the same domain for application and login can mitigate the impact of [third-party cookie blocking](azure/active-directory/develop/reference-third-party-cookies-spas) on an end user's authentication experience. 

Author: ```suggestion
-  Staying in the same domain for application and login can mitigate the impact of [third-party cookie blocking](/azure/active-directory/develop/reference-third-party-cookies-spas) on an end user's authentication experience. 
```

> -  Staying in the same domain for application and login can mitigate the impact of [third-party cookie blocking](/azure/active-directory/develop/reference-third-party-cookies-spas) on an end user's authentication experience. 

Reviewer: ```suggestion
-  By staying in the same domain for your application during sign-in, you mitigate the impact of [third-party cookie blocking](/azure/active-directory/develop/reference-third-party-cookies-spas). 
```
